lib/galaxy/jobs/__init__.py

```diff
@@ -1743,7 +1743,14 @@ def _change_ownership( self, username, gid ):
         job = self.get_job()
         # FIXME: hardcoded path
         external_chown_script = self.get_destination_configuration("external_chown_script", None)
-        cmd = [ '/usr/bin/sudo', '-E', external_chown_script, self.working_directory, username, str( gid ) ]
+
+        cmd = [ '/usr/bin/sudo', '-E', ]
```

Comment:
Hardcoded sudo is actually a remnant of the original implementation. Ideally, if one needs sudo it should just be part of the value of the corresponding `*_script` call, at which point you'd have full control over the environment variables passed or not passed.